src/open-response/components/runtime.tsx

```diff
@@ -33,6 +33,7 @@ export const Runtime: React.FC<IProps> = ({ authoredState, interactiveState, set
   const [audioSupported, setAudioSupported] = React.useState<boolean>(browserSupportsAudio);
   const [recordingStarted, setRecordingStarted] = React.useState<boolean>(false);
   const [recordingDisabled, setRecordingDisabled] = React.useState<boolean>(false);
+  const [recordingFailed, setRecordingFailed] = React.useState<boolean>(false);
   const [playDisabled, setPlayDisabled] = React.useState<boolean>(false);
   const [stopDisabled, setStopDisabled] = React.useState<boolean>(true);
```

Comment:
FYI, the `<boolean>` is unnecessary as TypeScript will infer the type correctly without it.

Reply:
@kswenson Good to know. Thanks 👍